package.json

```diff
@@ -34,7 +34,8 @@
     "format": "pnpm --silent skuba format",
     "lint": "pnpm --silent skuba lint && pnpm --silent lint:md",
     "lint:md": "remark --frail --quiet .",
-    "release": "pnpm --silent build && changeset publish",
+    "prepack": "pnpm --silent build",
```

Comment:
I thought that pnpm doesn't run `pre` scripts by default: https://pnpm.io/cli/run#differences-with-npm-run

We seem to be relying on these Changesets config options in our workflows:

https://github.com/seek-oss/skuba/blob/4049ce976c10b1002ace0339055684533c190882/.github/workflows/release.yml#L35-L38

https://github.com/seek-oss/skuba/blob/4049ce976c10b1002ace0339055684533c190882/.github/workflows/snapshot.yml#L31-L34

Reply:
https://pnpm.io/cli/publish#life-cycle-scripts

Seems to run before these. I figure it will only run a build if it needs to based on what's changed?